ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -95,17 +102,15 @@ Edit a queue type to modify its name, the number of workers assigned to it or th
 
 ### Edit the name or the execution priority
 
-Change the name of a queue type or the execution priority, for instance to ensure that certain actions will be executed first.
+Change the name of a queue type or the execution priority, for instance to ensure that certain actions will be executed first. Note that the *Default* queue type is predefined and cannot be edited.
```

Comment:
Note that the *Default* queue type is predefined and cannot be edited.

Ich finde es gut, dass dieser Satz hier prominenter ist. In Unterkapitel "Edit the number of workers and the throttling settings" ist aber im Schritt 1. Beide konsistent machen, entweder in Einführung oder im Schritt 1?

Reply:
Passt dort leider nicht